src/plugins/intel_cpu/src/utils/ngraph_utils.hpp

```diff
@@ -47,5 +49,28 @@ inline bool isDynamicNgraphNode(const std::shared_ptr<const ngraph::Node>& op) {
     return ret;
 }
 
+inline std::string get_port_name(const ov::Output<const ov::Node>& port, const bool is_legacy_api) {
+    std::string name;
+    // TODO: Should use tensor name as the port name, but many legacy tests still use legacy name
+    // plus sometimes it will get empty tensor name.
+    if (!is_legacy_api) {
+        name = {};
+        // auto tensor_ptr = port.get_tensor_ptr();
+        // if (tensor_ptr->get_names().size() >= 1) {
+        //     name = port.get_tensor_ptr()->get_any_name();
```

Comment:
It looks unsafe. `get_any_name()` may not return expected name.
Anyway please remove commented code.

Reply:
There are too many names in model/node, such as friendly name, unique name, legacy tensor name, tensor names, maybe we need a unified name for input/output port. So for legacy public api, we can still use `ov::op::util::get_ie_output_name` to get input/output name, and for new public API and plugin API, we need apply tensor name for input/output port.

Reply:
Have remove the commented code.